lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -74,6 +74,11 @@ protected function configure() : void
 
     <info>%command.full_name% FQCN</info>
 
+You can show more information about the process by increasing the verbosity level. To see the
+executed queries, set the level to debug with <comment>-vv</comment>:
```

Comment:
[The docs say debug level is obtained with `-vvv`](https://symfony.com/doc/current/console/verbosity.html)

Reply:
I decided to use the terminology used in the printed message, instead of the one used by Symfony. 

```
bobvandevijver@webdev:/var/www/dev$ bin/console doctrine:migrations:migrate -n -vv
[notice] Migrating up to DoctrineMigrations\Version20200618130453
[info] ++ migrating DoctrineMigrations\Version20200618130453
[debug] ALTER TABLE <snip>
[info] Migration DoctrineMigrations\Version20200618130453 migrated (took 180.7ms, used 22M memory)
[notice] finished in 199.9ms, used 22M memory, 1 migrations executed, 1 sql queries
```

Reply:
The actual call in the code is also `$this->logger->debug()`, so there might be some place where it is configured wrongly, or there is just simply a mismatch between the used terminology.

Reply:
> some place

That kind of configuration would be in your application though, there does not seem to be any code related to monolog in the bundle. You can use `bin/console debug:config monolog` to see what is up. [These docs](https://symfony.com/doc/current/logging/monolog_console.html) seem to talk about a mapping between log levels and verbosity, maybe that would be the root cause?

Reply:
I totally agreed with you concerning the application configuration. But as this is a clean project with only the default flex installed configuration in place for the loggin, I was sceptical. So I went digging.

It turns out not Monolog is used for the console output, but the [`ConsoleLogger`](https://github.com/doctrine/migrations/blob/f327c40fe06e30d68f4b3745ac88d0eded5e0777/lib/Doctrine/Migrations/Tools/Console/ConsoleLogger.php) from this library. And that clearly maps the debug loglevel to the very verbose output:

https://github.com/doctrine/migrations/blob/f327c40fe06e30d68f4b3745ac88d0eded5e0777/lib/Doctrine/Migrations/Tools/Console/ConsoleLogger.php#L47

Reply:
Excellent sleuth job! :detective:  So that's why the style is a bit different!